### PR TITLE
Supplement UI test for training a model

### PR DESCRIPTION
**Pull request by made-line**
https://github.com/code-dot-org/code-dot-org/pull/41527

### CODE REVIEW

File: src/UIComponents/DataTable.jsx [hunk 1]
```diff
@@ -134,6 +134,7 @@ class DataTable extends Component {
                   return (
                     <td
                       key={columnId}
+                      className="table-column"
```
Comment (breville) [inline src/UIComponents/DataTable.jsx:137]: Maybe `"data-table-column"` since we also use HTML tables in other places too?

Comment (made-line) [reply src/UIComponents/DataTable.jsx:137]: Sounds good!

**Review by Erin007 — COMMENTED** [review]
There's a convention in the main repo ([for example](https://github.com/code-dot-org/code-dot-org/search?q=uitest)) to prepend `uitest` to selectors that are used specifically for UI tests. Save future folks from going hunting for css and it's easier to search. It'd be good to be consistent here. 

Otherwise, excited to see this work coming along! Yay for test coverage!

**Comment by made-line** [comment]
@Erin007 Thanks for pointing that out!

**Review by Erin007 — APPROVED** [review]

**Pull request merged**